.github/workflows/ci.yml

```diff
@@ -6,50 +6,44 @@ on:
   pull_request:
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ (matrix.arch == '') && 'default' || matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         version:
-          - '1.0'
-          - '1.6'
+          - 'min'
+          - 'lts'
           - '1'
-          - nightly
+          - 'pre'
         os:
-          - ubuntu-latest
+          - ubuntu-latest # Apple Silicon
```

Comment:
Did you mean to have this comment on this line or the following one?

Reply:
Ah good catch, of course it should be moved to the following line.